src/devices/Nmea0183/tests/Ais/AisManagerTest.cs

```diff
@@ -0,0 +1,415 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using Iot.Device.Common;
+using Iot.Device.Nmea0183.Ais;
+using Iot.Device.Nmea0183.Sentences;
+using Microsoft.VisualStudio.CodeCoverage;
+using Shouldly;
+using UnitsNet;
+using Xunit;
+using NavigationStatus = Iot.Device.Nmea0183.Ais.NavigationStatus;
+
+namespace Iot.Device.Nmea0183.Tests.Ais
+{
+    public class AisManagerTest : IDisposable
+    {
+        private readonly AisManager _manager;
+
+        public AisManagerTest()
+        {
+            _manager = new AisManager("Test", true, 269110660u, "Cirrus");
+        }
+
+        public void Dispose()
+        {
+            _manager.Dispose();
+        }
+
+        [Fact]
+        public void Initialisation()
+        {
+            _manager.TrackEstimationParameters.MaximumPositionAge.ShouldBeEquivalentTo(TimeSpan.FromSeconds(20));
+            _manager.DimensionToStern.ShouldBeEquivalentTo(Length.Zero);
+            _manager.OwnMmsi.ShouldBeEquivalentTo(269110660u);
+            _manager.OwnShipName.ShouldNotBeEmpty();
+        }
+
+        [Fact]
+        public void FeedWithRealDataAndCheckGeneralAttributes()
+        {
+            using NmeaLogDataReader reader = new NmeaLogDataReader("Reader", "../../../Nmea-2021-08-25-16-25.txt");
```

Comment:
We should probably copy this file to output directory, we should not try to guess the paths with `..`

Reply:
True, I was just to lazy for that...

Now embedded the test data as resources, to avoid this problem.